shared/styles/navbar.css

```diff
@@ -7,6 +7,7 @@
   background: linear-gradient(180deg, lavenderblush 0%, white 100%);
   box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 10%);
   position: relative;
+  z-index: 2;
```

Comment:
there should not be an overlap between the layers. I don't think there is a need to increase the z-index here.